markets/bfp-market/contracts/utils/ErrorUtil.sol

```diff
@@ -124,4 +124,7 @@ library ErrorUtil {
 
     /// @notice Thrown when user trying to merge accounts with positions on opposite sides.
     error InvalidPositionSide();
+
+    /// @notice Thrown when user trying commit an order with `keeperFeeBufferUsd` larger than margin.
+    error KeeperBufferFeeTooLarge(uint256 keeperFeeBufferUsd, uint256 max);
```

Comment:
`max` is not technically the max as it's more like `maxKeeperFee - fees`, I'd just revert here without any of the attributes as (1) `keeperBufferUsd` is already supplied on commit, (2) if it's failing on settlement then we already store in the order, and (3) max here isn't the true max.